SandboxUwp/MainPage.xaml.cs

```diff
@@ -72,6 +72,32 @@ public MainPage()
             OutputBlock.Text = "Created blood pressure.";
         }
 
+        private async void Get_Height_OnClick(object sender, RoutedEventArgs e)
+        {
+            PersonInfo personInfo = await connection.GetPersonInfoAsync();
+            HealthRecordInfo recordInfo = personInfo.SelectedRecord;
+            IThingClient thingClient = connection.CreateThingClient();
+            var heights = await thingClient.GetThingsAsync<Height>(recordInfo.Id);
+            Height firstHeight = heights.FirstOrDefault();
+            if (firstHeight == null)
+            {
+                OutputBlock.Text = "No height.";
+            }
+            else
+            {
+                OutputBlock.Text = firstHeight.Value.Meters.ToString();
```

Comment:
here is more info abt formatting -  [https://msdn.microsoft.com/en-us/library/system.string.format(v=vs.100).aspx](https://msdn.microsoft.com/en-us/library/system.string.format(v=vs.100).aspx)

---
In reply to: [118126758](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/40#discussion_r118126758) [](ancestors = 118126758)